doc/developer/workflow.rst

```diff
@@ -108,7 +108,41 @@ March/July/November.  Walking backwards from this date:
    (The :file:`configure.ac` edit and tag push are considered git housekeeping
    and are pushed directly to ``master``, not through a PR.)
 
- - 2 weeks earlier, a ``rc2`` release candidate is tagged.
+   Below is the snippet of the commands to use in this step.
+
+     .. code-block:: console
+
+        % git remote --verbose
+        origin    git@github.com:ton31337/frr (fetch)
+        origin    git@github.com:ton31337/frr (push)
+        upstream  https://github.com/frrouting/frr (fetch)
+        upstream  https://github.com/frrouting/frr (push)
+
+        % git checkout master
+        % git pull upstream master
+        % git checkout -b dev/8.2
+        % git tag base_8.2
+        % git push upstream base_8.2
+        % git push upstream dev/8.2
+        % git checkout master
+        % sed -i 's/8.2-dev/8.3-dev/' configure.ac
+        % git tag -a frr-8.3-dev -m "frr-8.3-dev"
+        % git push upstream frr-8.3-dev
+        % git add configure.ac
+        % git commit -s -m "build: FRR 8.3 development version"
+        % git push upstream master
+
+   In this step, we also have to update package versions to reflect
+   the development version. Versions need to be updated using
+   a standard way of development (Pull Requests) based on master branch.
+
+   Only change the version number with no other changes. This will produce
+   packages with the new version number that is height than any previous
+   version. Once the release is done, whatever changes we make the changelog
+   needs to be cherry picked from the release branch (``stable/X.Y``)
```

Comment:
wording : whatever updates we make to changelog files on the release branch need to be cherry-picked to the master branch.

Reply:
Fixed.